validator/validator-main.protoascii

```diff
@@ -3385,6 +3385,30 @@ attr_lists: {
   }
 }
 
+# Video Interface attributes for AMP. These attributes can be attached to any
+# element that implements the video interface
+attr_lists: {
+  name: "implements-video-interface-AMP"
+  attrs: {
+    name: "autoplay"
+    value: ""
+  }
+  attrs: {
+    name: "fullscreen-on-landscape"
+    value: ""
+  }
+}
+
+# Video Interface attributes for A4A. These attributes can be attached to any
+# element that implements the video interface
+attr_lists: {
+  name: "implements-video-interface-A4A"
```

Comment:
These names should be lowercase throughout. Please change to `name: "implements-video-interface-amp4ads"`.